packages/frontend/editor-ui/src/views/SettingsSso.test.ts

```diff
@@ -82,6 +82,8 @@ describe('SettingsSso View', () => {
 
 			const { getByTestId } = renderView();
 
+			expect(() => getByTestId('sso-auth-protocol-select')).toThrow('Unable to find an element by');
+
```

Comment:
[nitpick] https://testing-library.com/docs/guide-disappearance/#asserting-elements-are-not-present

the correct way of testing the non existence of an element would be

 ```suggestion
			const { getByTestId, queryByTestId } = renderView();

			expect(queryByTestId('sso-auth-protocol-select')).not.toBeInTheDocument();

```